### DISCUSSION TITLE
Voice Controlled Robot (Project 30)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#7] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #30](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Voice%20Controlled%20Robot)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by swiicoder** [comment]
Hi, I'm interested in this project and I wanted to ask a few questions. First, does this project require a start and an end date? Second, is there a team requirement? And, finally can additional people contribute to this project once it has been undertaken?

**Reply by robertogl** [reply]
Hi there,

Short answer to the first question is no! However, we normally consider as starting data the date you fill out the form, which link you can find at the top of each project's page, but that is just for our records. We do not give you a deadline, just submit your solution as soon as it is ready. You will receive more information about how to submit the solution in an email that is sent to you right after you fill out the sign up form. 

You can work on a project alone and additional people can certainly contribute to the project. Eventually, If you think they provided enough contribution, we will issue a certificate to them as well. 

I hope this helps! If you have more questions do not hesitate to ask!

Cheers,
Roberto

**Reply by swiicoder** [reply]
Thank you for the information.

**Comment by ghost** [comment]
Hello everyone I'm interested in this project and ant to  contribute in it!!

**Comment by MayankGargtmc** [comment]
Hello, We too interested in this project , have made our team already!
All the to you'll

**Comment by ghost** [comment]
Excited to join the team, waiting for further responses

**Comment by pieeejing** [comment]
Hi, my name is Jing, and I am a student at UBC-V. Excited to work on this project and hopefully come up with a solution!

**Comment by neharika950** [comment]
hello,
Do we need hardware to do the project or online simulations like tinkercad and proteus would qualify for the rewards?

**Reply by robertogl** [reply]
Simulation is perfectly fine!

**Comment by Praveena22399** [comment]
Hi, I am a student at cochin university of science and technology and interested in working and collaborating on this project
